index.html

```diff
@@ -10,7 +24,64 @@ <h3>Heading 3</h3>
 <h4>Heading 4</h4>
 <h5>Heading 5</h5>
 <h6>Heading 6</h6>
-<hr />
+<hr>
+<div class="home">
+  <h1 id="content" class="post-list-heading"> <svg class="svg-icon" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+      <path d="M19 21l-7-5-7 5V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2z"></path>
+    </svg> Home Heading 1 with icon</h1>
```

Comment:
```suggestion
  <h1 id="content" class="post-list-heading">{% include icon/bookmark.svg %} Home Heading 1 with icon</h1>
```

Reply:
Oops. That's what I get for copying DOM nodes out of dev tools! thanks